app/src/main/java/com/ciandt/dragonfly/example/components/chips/ChipAdapter.kt

```diff
@@ -25,7 +25,7 @@ class ChipAdapter(var context: Context, var list: ArrayList<out Chip>, val onCli
 
         val view = LayoutInflater.from(context).inflate(layout, parent, false)
 
-        view.findViewById(R.id.chipButton) as Button? ?: throw IllegalArgumentException("Layout for item should contain a Button with id = button")
+        view.findViewById<Button?>(R.id.chipButton) ?: throw IllegalArgumentException("Layout for item should contain a Button with id = button")
```

Comment:
User informs the layout for chip by an attribute: `chipsView:chipLayout="@layout/bla_bla_bla"` and this layout should have a button with id = chipButton.

When we check this, we return this:
`java.lang.IllegalArgumentException: Layout for item should contain a Button with id = chipButton`

If we dont, the result is:
`java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference`